io.mpv.Mpv.appdata.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2012-2019 mpv.io -->
+<component type="desktop">
+ <id>io.mpv.Mpv</id>
+ <metadata_license>CC-BY-SA-4.0</metadata_license>
+ <project_license>GPL-2.0+ AND LGPL-2.0-or-later</project_license>
+ <name>mpv</name>
+ <summary>a free, open source, and cross-platform media player</summary>
+ <launchable type="desktop-id">io.mpv.Mpv.desktop</launchable>
```

Comment:
Is the desktop file correctly renamed by flatpak-builder? It seems to be called `mpv.desktop` in upstream.